config_defaults_inc.php

```diff
@@ -2066,6 +2066,13 @@
  */
 $g_wrap_in_preformatted_text = ON;
 
+/**
+ * Controls whether attachment links redirect users t a new tab or not
+ * If turned off, a click on a link will not redirect to a new tab
+ * @global integer $g_attachment_to_new_tab
+ */
+$g_attachment_to_new_tab = OFF;
+
```

Comment:
I believe it would make more sense to move this to the *Bug Attachments Settings* section